cypress/integration/sales/credit_memo_for_sales_invoice.js

```diff
@@ -1,23 +1,46 @@
+/*
+ * #%L
+ * metasfresh-pharma
```

Comment:
```suggestion
 * metasfresh-e2e
```

good idea to add the licence header, but pls add `metasfresh-e2e`, and the year 2019 (instead of 2018)